core/src/datayoga_core/job.py

```diff
@@ -125,6 +125,49 @@ def transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> Job
 
         return result
 
+    async def async_transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> JobResult:
```

Comment:
can we have test(s) to cover this new functionality?

Reply:
There is no new logic. Current tests must be refactored to async as soon as we'll finish refactor.